src/components/menu/menu-link/menu-link.ts

```diff
@@ -0,0 +1,76 @@
+import { spread } from '@open-wc/lit-helpers';
+import { nothing, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { html, unsafeStatic } from 'lit/static-html.js';
+
+import {
+  LanguageController,
+  type LinkRenderVariables,
+  SbbDisabledMixin,
+  SbbIconNameMixin,
+  resolveLinkOrStaticRenderVariables,
+  SbbLinkBaseElement,
+  targetsNewWindow,
+} from '../../core/common-behaviors';
+import { setAttributes } from '../../core/dom';
+import { i18nTargetOpensInNewWindow } from '../../core/i18n';
+import '../../icon';
+import { SbbMenuActionCommonElementMixin } from '../common/menu-action-common';
+
+/**
+ * It displays a link element that can be used in the `sbb-menu` component.
+ *
+ * @slot - Use the unnamed slot to add content to the `sbb-menu-link`.
+ * @slot icon - Use this slot to provide an icon. If `icon-name` is set, a `sbb-icon` will be used.
+ */
+@customElement('sbb-menu-link')
+export class SbbMenuLinkElement extends SbbMenuActionCommonElementMixin(
+  SbbIconNameMixin(SbbDisabledMixin(SbbLinkBaseElement)),
+) {
+  private _language = new LanguageController(this);
+
+  protected override render(): TemplateResult {
+    const {
+      tagName: TAG_NAME,
+      attributes,
+      hostAttributes,
+    }: LinkRenderVariables = resolveLinkOrStaticRenderVariables(this);
+    setAttributes(this, hostAttributes);
+
+    /* eslint-disable lit/binding-positions */
+    return html`
+      <${unsafeStatic(TAG_NAME)} class="sbb-menu-action" ${spread(attributes)}>
```

Comment:
I think we can remove the static variant for the menu button/link.

Reply:
Done in [9d4d702](https://github.com/lyne-design-system/lyne-components/pull/2364/commits/9d4d702b14963775aa5b8c002d1a9cd153f70c3f)